code/modules/jobs/job_types/roguetown/church/mortician.dm

```diff
@@ -14,7 +14,7 @@
 	outfit = /datum/outfit/job/roguetown/undertaker
 	display_order = JDO_GRAVEMAN
 	give_bank_account = TRUE
-	min_pq = -5
+	min_pq = 15
```

Comment:
15 is excessive, 5 should be enough to prevent farming out of negative PQ for it.